common/repositories.sh

```diff
@@ -45,17 +46,20 @@ sawtooth_repositories=(
   https://github.com/hyperledger/sawtooth-seth.git
   https://github.com/hyperledger/sawtooth-supply-chain.git
   https://github.com/hyperledger/sawtooth-website.git
-  https://github.com/hyperledger/education-sawtooth-simple-supply.git
 )
 
 iroha_repositories=(
-  https://github.com/hyperledger/iroha.git
   https://github.com/hyperledger/iroha-android.git
   https://github.com/hyperledger/iroha-dotnet.git
   https://github.com/hyperledger/iroha-ed25519.git
   https://github.com/hyperledger/iroha-ios.git
+  https://github.com/hyperledger/iroha-java.git
   https://github.com/hyperledger/iroha-javascript.git
+  https://github.com/hyperledger/iroha-network-tools.git
```

Comment:
This says ARCHIVED

Reply:
I transferred it to the archive and updated this line